platform/server/src/main/java/org/motechproject/server/impl/JspBundleLoader.java

```diff
@@ -204,7 +206,19 @@ private static void clearTomcatCache() throws InvocationTargetException, NoSuchM
         Class cl = loader.getClass();
 
         if ("org.apache.catalina.loader.WebappClassLoader".equals(cl.getName())) {
-            clearMap(cl, loader, "resourceEntries");
+            ArrayList<Field> fields = new ArrayList<Field>(Arrays.asList(cl.getDeclaredFields()));
```

Comment:
why do we need the array list? and the list from arrays? why not just iterate over the array?

Anyway, probably would be best to just use FieldUtils.getDeclaredField() from apache commons-lang.